src/interfaces/ISablierMerkleBase.sol

```diff
@@ -113,21 +114,21 @@ interface ISablierMerkleBase is IAdminable {
     /// @param amount The amount of tokens to claw back.
     function clawback(address to, uint128 amount) external;
 
-    /// @notice Collects the accrued fees by transferring them to {FACTORY}.
+    /// @notice Collects the accrued fees by transferring them to the `to` address.
     ///
     /// Requirements:
     /// - `msg.sender` must be {FACTORY}.
     ///
-    /// @param factoryAdmin The address of the admin of {FACTORY}.
+    /// @param to The address to receive the fees.
     /// @return feeAmount The amount of native tokens (e.g., ETH) collected as fees.
-    function collectFees(address factoryAdmin) external returns (uint256 feeAmount);
+    function collectFees(address to) external returns (uint256 feeAmount);
 
     /// @notice Lowers the min USD fee.
     ///
     /// @dev Emits a {LowerMinFeeUSD} event.
     ///
     /// Requirements:
-    /// - `msg.sender` must be the admin of {FACTORY}.
+    /// - `msg.sender` must have {FEE_MANAGEMENT_ROLE} role in {FACTORY}.
```

Comment:
RBAC doesn't exist yet

Reply:
I thought I removed it from everywhere. But missed this one.